directory/datadog_checks/directory/directory.py

```diff
@@ -44,13 +44,15 @@ def __init__(self, *args, **kwargs):
         self.config = DirectoryConfig(self.instance)
 
     def check(self, _):
+        service_check_tags = ['dir_name:{}'.format(self.config.name)]
```

Comment:
Should it be `dirtagname`?

Reply:
The key of the tag is fetched from the configuration. We could do that, but then we wouldn't be able to auto-group the service checks in monitores (the "groups" field in service_checks.json), as the tag key wouldn't be deterministic